portfolio/pom.xml

```diff
@@ -33,6 +33,11 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-translate</artifactId>
+      <version>1.70.0</version>
+    </dependency>
```

Comment:
Ignore this commit, I accidentally added it to this branch

Reply:
You should be able to revert this file using git revert.